docs/content-server-api/pointerChanges/get.md

```diff
@@ -0,0 +1,76 @@
+# Get Pointer Changes
+
+List all deployment changes from the same pointer
+
+**URL** : `/pointerChanges`
+
+<!-- **URL Parameters** : `pk=[integer]` where `pk` is the ID of the Account on the server. -->
```

Comment:
What's this?

Reply:
It's the template for documenting URL params, I'm deleting it as we are not using it.